src/Forms/ConfigurationForm.cs

```diff
@@ -16,14 +16,26 @@ public ConfigurationForm(AzureMigrateExportMainForm obj)
         {
             InitializeComponent();
             mainFormObj = obj;
+
+            // Set the DropDownStyle to DropDownList to disable text entry
+            ModuleComboBox.DropDownStyle = ComboBoxStyle.DropDownList;

```

Comment:
Why was this required?

Reply:
because in the custom workflow module box, ideally a user (rather than choosing between the 2 options) shouldn't be allowed to type his own workflow and then receive an error.